test/conv_common.hpp

```diff
@@ -1800,7 +1800,7 @@ struct conv_driver : test_driver
         // by default, this member is constructed when conv2d/3d is constructed (see
         // test_driver::add())
         // but this requires the dimensions come from commandline, which is hard for non-NCHW layout
-        if(in_layout != "NCHW" || in_layout != "NCDHW")
+        if(in_layout != "NCHW" && in_layout != "NCDHW")
```

Comment:
@carlushuang Please notice this mistake. The code looked suspicious to me at the review time, but I missed this issue ;) https://github.com/ROCmSoftwarePlatform/MIOpen/pull/765/files#r601858944

Reply:
Nice catch!